lldb/test/API/functionalities/scripted_process/invalid_scripted_process.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, exe_ctx: lldb.SBExecutionContext, args: lldb.SBStructuredData
     def read_memory_at_address(
         self, addr: int, size: int, error: lldb.SBError
     ) -> lldb.SBData:
-        error.SetErrorString("This is an invalid scripted process!")
+        error = Status::FromErrorString("This is an invalid scripted process!")
```

Comment:
This is not valid python. The regex should only be applied to C++.

Reply:
Thanks!